packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/simulation/Simulation.java

```diff
@@ -72,31 +68,38 @@ public class Simulation implements Serializable {
 	@Enumerated(EnumType.STRING)
 	private SimulationEngine engine;
 
-	@JsonAlias("workflow_id")
-	private UUID workflowId;
+	@OneToOne
+	@JoinColumn(name = "workflow_id", nullable = true)
+	@JsonBackReference
+	private Workflow workflow;
 
 	@JsonAlias("user_id")
 	@TSOptional
 	private String userId;
 
 	@JsonAlias("project_id")
 	@TSOptional
-	private UUID projectId;
+	private UUID projectId; // TODO this can probably be joined to the project table soon?
```

Comment:
Do you mean via `ProjectAsset`?

Reply:
`ProjectAsset` may be able to go away... they were stubs holding these things in postgres!